subsys/bluetooth/services/fast_pair/Kconfig.fast_pair

```diff
@@ -184,6 +186,36 @@ config BT_FAST_PAIR_STORAGE_INTEGRATION_INIT_PRIORITY
 
 endif # BT_FAST_PAIR_STORAGE_INTEGRATION
 
+config BT_FAST_PAIR_BOND_MANAGER
+	bool "Fast Pair bond management functionality [EXPERIMENTAL]"
+	default y
+	depends on !BT_FAST_PAIR_FMDN
+	select EXPERIMENTAL
+	select BT_FAST_PAIR_STORAGE_AK_BOND
+	help
+	  If this option is enabled, the Fast Pair subsystem tracks the Bluetooth bonds created
+	  through the Fast Pair Procedure and unpairs them if the Fast Pair Procedure is incomplete
+	  or the Account Key associated with the bonds is removed. It also unpairs the Fast Pair
+	  Bluetooth bonds on Fast Pair factory reset. Keep in mind that when the Fast Pair subsystem
+	  is disabled, the Bluetooth bonds are not tracked. Do not perform operations on the Fast
+	  Pair Bluetooth bonds when the Fast Pair subsystem is disabled.
+	  When this option is enabled, non-volatile storage is used to store the bond information.
+	  You can enable this option during DFU, but you cannot disable it during DFU if you used it
+	  earlier unless you erase whole Fast Pair non-volatile storage. Otherwise, the Fast Pair
+	  enable operation will fail.
+
+if BT_FAST_PAIR_BOND_MANAGER
+
+config BT_FAST_PAIR_BOND_MANAGER_INIT_PRIORITY
+	int
+	range 0 9
+	default 3
+	help
+	  Fast Pair bond manager module initialization priority.
```

Comment:
Do we still need to use lower value for initialization priority here? The callbacks registration that enforced initialization order is now moved to `SYS_INIT`